app/res/values/colors.xml

```diff
@@ -4,6 +4,7 @@
     <color name="transparent">#00000000</color>
     <color name="black">#000000</color>
     <color name="grey_darker">#444444</color>
+    <color name="grey_medium">#616161</color>
```

Comment:
What do we use this for? 

Biyeun wanted to reduce the total set of colors we use in the app to give things a common feel. Might be worth consulting on where this is used and what it's for.